app/views/checkout/payment.scala.html

```diff
@@ -21,15 +21,19 @@
         <span class="option__input">
             <input
             type="radio" name="ratePlanId"
+            data-option-mirror-label-default="@plan.prettyPricing(currency)"
             data-option-mirror-label="@plan.prettyPricing(currency)"
-            data-amount="@plan.pricing.getPrice(currency).get.amount"
```

Comment:
This was the analytics bug which I fixed, where the price was getting set to the currency-applied amount but the analytics library is not sent the currency. So I've forced it to be gbpPrice, as is what's used in the other pages of the digipack flow.